src/components/atoms/button/button.scss

```
@@ -0,0 +1,27 @@
+@use "../../variables" as *;
+
+.button {
+  //スタイル
+  width: 200px;
+  height: 55px;
+  background: $black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: $white;
+  transition: ease 0.2s;
+  font-size: 14px;
```

```suggestion
  font-size: $fontsize-s;
```
フォントサイズは変数を使うようにしましょう！